types/blobshape/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for blobshape 1.0
+// Project: https://github.com/lokesh-coder/blobshape#readme
+// Definitions by: OyewoleOyedeji <https://github.com/OyewoleOyedeji>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface BlobshapeParameters {
+    size?: number;
+    growth?: number;
+    edges?: number;
+    seed?: null;
+}
+
+export default function blobshape({ size, growth, edges, seed }?: BlobshapeParameters): {
```

Comment:
you've probably need CommonJS export pattern for this module:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

so probably something along:
```ts
declare function generator({ size, growth, edges, seed }?: generator.BlobshapeParameters): {
    path: string;
    seedValue: number;
};

declare namespace generator {
    interface BlobshapeParameters {
        size?: number;
        growth?: number;
        edges?: number;
        seed?: null;
    }
}

export = generator;
```

Reply:
The package is an ES module